README.md

```diff
@@ -72,34 +72,35 @@ You must have ssh access to all the nodes. You also need root :)
 
    c. Helm for installing software on to the cluster.
 
-   d. An nginx ingress that is installed on all nodes - this is used to get
-      network traffic into the cluster. This is installed via helm.
+   d. An Nginx ingress that is installed on all nodes - this is used to get
+      network traffic into the cluster. This is installed via Helm.
 
-   e. Credentials to access the kubernetes cluster in the currently running user's
+   e. Credentials to access the Kubernetes cluster in the currently running user's
       `~/.kube/config` directory.
 
    The master node is also marked as schedulable - this might not be ideal if
    you are running a large cluster, but is useful otherwise. This also means
    that if you only wanted a single node Kubernetes cluster, you are already
    done!
    
-5. Test that everything is up!
+3. Test that everything is up!
 
    a. Run `kubectl get node` - you should see one node (your master node) marked
       as `Ready`.
 
    b. Run `kubectl --namespace=kube-system get pod`. Everything should be in
-      `Running` state. If it's still `Pending`, give it a couple minutes. If
-       they are in `Error` or `CrashLoopBackoff` state, something is wrong.
+      `Running` state.  If it's still `Pending`, give it a couple minutes. If
+       they are in `Error` or `CrashLoopBackoff` state, something is wrong. Here's an example of what it might look like:
```

Comment:
Is there an example snippet missing here?

Reply:
Oops.  I put an example in, but then took it out because I couldn't figure out how to get a code snippet to indent correctly for enumerated list, so I took it back out.   I'll fix that.